src/main/app/claims/models/claim.ts

```diff
@@ -34,4 +39,12 @@ export default class Claim implements Serializable<Claim> {
   get remainingDays (): number {
     return this.responseDeadline.diff(MomentFactory.currentDate(), 'days')
   }
+
+  private deserializeDocuments (documents: any): UploadedDocument[] {
+    if (documents) {
+      return documents.map((document: any) => {
```

Comment:
`return documents.map((document) => new UploadedDocument().deserialize(document)` would be enough.

Reply:
done